skins/moono-lisa/dialog.css

```diff
@@ -1121,7 +1110,7 @@ a.cke_specialchar:active
 }
 
 .cke_dialog_image_url .cke_dialog_ui_hbox_last,
-.cke_dialog_flash_url .cke_dialog_ui_hbox_last
+.cke_dialog_ui_hbox_last
 {
 	vertical-align: bottom;
 }
```

Comment:
Previously this CSS rule was changing styling of `.cke_dialog_ui_hbox_last` as a child of `.cke_dialog_flash_url` but right now it will be used for every element outside flash? If so, the whole rule should be rather removed.